docs/pulseload.rst

```diff
@@ -10,7 +10,7 @@ posting your own data.
 Configuration
 -------------
 
-Specify the exchanges to read from using environment variables. For example::
+Specify the exchanges to read from in settings.py. A working example::
```

Comment:
Actually, we should be encouraging people to set these values in their environment, rather than in ``settings.py``.  Since ``settings.py`` is checked in, making direct changes will end up being a hassle with their PR.  

But with the changes I suggested for ``settings.py`` we could tell them that the default will get everything from TC, so they may not need to set anything.  Otherwise, they can customize it in their env with ``export`` in bash.

Reply:
In line 15 that sets ``PULSE_DATA_INGESTION_SOURCES`` we can describe how they can customize the setting, but make it a ``bash`` line with ``export`` instead of this chunk of json.  Something like:

    export PULSE_DATA_INGESTION_SOURCES='[{"exchange": "exchange/taskcluster-treeherder/v1/jobs", "destinations": ["#"], "projects": ["#"]}]'

So, describe something like, "``projects`` can be set to just "mozilla-inbound" if you only want jobs for that one repository."  Something along those lines.

Reply:
Done. Hopefully correctly.